src/workerd/api/global-scope.c++

```diff
@@ -690,14 +690,14 @@ void ServiceWorkerGlobalScope::queueMicrotask(jsg::Lock& js, v8::Local<v8::Funct
           (jsg::Lock & js, const v8::FunctionCallbackInfo<v8::Value>& args)->v8::Local<v8::Value> {
             return js.tryCatch([&]() -> v8::Local<v8::Value> {
               auto function = fn.getHandle(js);
-              auto context = js.v8Context();
 
-              kj::Vector<v8::Local<v8::Value>> argv(args.Length());
+              v8::LocalVector<v8::Value> argv(js.v8Isolate, args.Length());
```

Comment:
Nit: Can we have a helper function to convert `FunctionCallbackInfo<T>` to `LocalVector<T>`? We can add a todo to simplify it once `FunctionCallbackInfo` has iterator support?